recipe/meta.yaml

```diff
@@ -36,9 +35,8 @@ requirements:
     - r-base
     - r-rcpp >=1.0.5
   run:
+    - libwinpthread  # [win]
```

Comment:
Shouldn't be necessary - it's already an implicit run_exports.

```suggestion
```

Reply:
As it currently stands, if I don't add this, it fails the linking audit.  I'm going to try adding it to `host`.  If that doesn't work then there may be an issue with the run export for this - this is also not the only package where this issue came up.

Reply:
It seems `libwinpthread` does not have a proper run export, because I get the linking error if I put `libwinpthread` in `host`:
```
  ERROR (r-rlemon,lib/R/library/rlemon/libs/x64/rlemon.dll): Needed DSO Library/bin/libwinpthread-1.dll found in ['conda-forge/win-64::libwinpthread==12.0.0.r4.gg4f2fc60ca=h57928b3_8']
  ERROR (r-rlemon,lib/R/library/rlemon/libs/x64/rlemon.dll): .. but ['conda-forge/win-64::libwinpthread==12.0.0.r4.gg4f2fc60ca=h57928b3_8'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
```
Indeed, in the feedstock the recipe doesn't have `run_exports` defined: https://github.com/conda-forge/winpthreads-feedstock/blob/main/recipe/meta.yaml.

Should we may be put `libwinpthread` back in `run` here for now so as not to block the migration and then open an issue in the `winpthread` feedstock?

Reply:
Never mind, it turns out I needed `winpthreads-devel` in `host`, and that sets the run export correctly for `libwinpthread`.  We should be all set here now.